src/styles/searchResults.css

```diff
@@ -0,0 +1,26 @@
+.results {
```

Comment:
It would be good if this class could be renamed to `.search-results` so it matches up with expectation, plus would make it easier to debug should it be part of a bigger app.

Also, CSS files are most often named in kebab-case, in part to help differentiate them from JS/JSX files at glance